tests/online_migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -59,12 +59,19 @@ sub run {
 
     # giva a little time to check package conflicts
     if (check_screen("yast2-migration-conflicts", 15)) {
-        send_key_until_needlematch 'migration-proposal-packages', 'tab', 3;
-        send_key "ret";
+        if (get_var("DESKTOP") =~ /textmode|minimalx/) {
+            send_key "alt-c";
+            send_key "alt-p";    # show package dependencies
+        }
+        else {
+            assert_and_click 'migration-proposal-packages';
+        }
```

Comment:
I don't really understand why you want to work around these dependency issues. Shouldn't the system be capable of working as is? *If* this is a workaround for a known issue with reported bug on bugzilla, please record a soft failure with reference to that bug.